app/gcl/associate-cloud-engineer/complete-advanced-guide/page.css

```
@@ -0,0 +1,801 @@
+                background: var(--accent-glow);
+                border-left-color: var(--gcp-blue);}
+.complete-guide-page .nav-item.active {
+                color: var(--gcp-blue);
+                background: rgba(26, 115, 232, 0.08);
+                border-left-color: var(--gcp-blue);
+                font-weight: 500;}
+.complete-guide-page .nav-item .nav-dot {
+                width: 6px;
+                height: 6px;
+                border-radius: 50%;
+                background: currentColor;
```

Stylelint (17.12.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)